applications/IgaApplication/custom_conditions/sbm_neumann_laplacian_condition.h

```diff
@@ -0,0 +1,271 @@
+    {};
+
+    /// Default constructor
+    SbmNeumannLaplacianCondition() : Condition()
+    {};
+
+    /// Destructor
+    virtual ~SbmNeumannLaplacianCondition() override
+    {};
+
+    ///@}
+    ///@name Life Cycle
+    ///@{
+
+    /// Create with Id, pointer to geometry and pointer to property
+    Condition::Pointer Create(
+        IndexType NewId,
+        GeometryType::Pointer pGeom,
+        PropertiesType::Pointer pProperties
+    ) const override
+    {
+        return Kratos::make_intrusive<SbmNeumannLaplacianCondition>(
+            NewId, pGeom, pProperties);
+    };
+
+    /// Create with Id, pointer to geometry and pointer to property
+    Condition::Pointer Create(
+        IndexType NewId,
+        NodesArrayType const& ThisNodes,
+        PropertiesType::Pointer pProperties
+    ) const override
+    {
+        return Kratos::make_intrusive<SbmNeumannLaplacianCondition>(
+            NewId, GetGeometry().Create(ThisNodes), pProperties);
+    };
+
+    ///@}
+    ///@name Operations
+    ///@{
+
+
+    void Initialize(const ProcessInfo& rCurrentProcessInfo) override;
+
+    /**
+    * @brief This is called during the assembling process in order
+    *        to calculate the condition right hand side matrix
+    * @param rLeftHandSideMatrix the condition right hand side matrix
+    * @param rCurrentProcessInfo the current process info
+    */
+    void CalculateRightHandSide(
+        VectorType& rRightHandSideVector,
+        const ProcessInfo& rCurrentProcessInfo) override;
+
+    /**
+    * @brief This is called during the assembling process in order
+    *        to calculate the condition left hand side matrix
+    * @param rLeftHandSideMatrix the condition left hand side matrix
+    * @param rCurrentProcessInfo the current process info
+    */
+    void CalculateLeftHandSide(
+        MatrixType& rLeftHandSideMatrix,
+        const ProcessInfo& rCurrentProcessInfo) override;
+
+    /**
+     * @brief This function provides a more general interface to the element.
+     * @details It is designed so that rLHSvariables and rRHSvariables are
+     *          passed to the element thus telling what is the desired output
+     * @param rLeftHandSideMatrix container with the output Left Hand Side matrix
+     * @param rRightHandSideVector container for the desired RHS output
+     * @param rCurrentProcessInfo the current process info instance
+     */
+    void CalculateLocalSystem(
+        MatrixType& rLeftHandSideMatrix,
+        VectorType& rRightHandSideVector,
+        const ProcessInfo& rCurrentProcessInfo) override;
+    /**
+    * @brief Sets on rResult the ID's of the element degrees of freedom
+    * @param rResult The vector containing the equation id
+    * @param rCurrentProcessInfo The current process info instance
+    */
+    void EquationIdVector(
+        EquationIdVectorType& rResult,
+        const ProcessInfo& rCurrentProcessInfo
+    ) const override;

```

Comment:
```suggestion
        const ProcessInfo& rCurrentProcessInfo) const override;
```
(just to be consistent with the rest of the file)